File: backend/Models/Dto/UserDto.cs
public class UserDto
{
  public int Id { get; set; }
  public string Email { get; set; } = string.Empty;
  public string FirstName { get; set; } = string.Empty;
  public string LastName { get; set; } = string.Empty;
  public List<string> Specialization { get; set; } = new List<string>();
  public string Skills { get; set; } = string.Empty;
}

File: saas-platform/src/app/features/auth/models/user-dto.ts
import { SpecializationType } from "./specialization-type.enum";
import { RoleType } from "./role-type.enum";


export interface CurrentUserDto {
  id: number,
  email: string,
  role: RoleType,
  firstName: string,
  lastName: string,
  nickname: string,
  description: string,
  specialization: SpecializationType[],
  companyName: string | null,
  companyNip: string | null,
  createdAt: Date,
  isActive: boolean
  permissions: string[]
  roles: string[]
}
